src/dom.js

```diff
@@ -912,3 +912,14 @@ export function toggleAttribute(element, name, forced) {
 
   return enabled;
 }
+
+/**
+ * @param {!Window} win
+ * @return {number} The width of the vertical scrollbar, in pixels.
+ */
+export function getVerticalScrollbarWidth(win) {
```

Comment:
Does this work cross browser?

Reply:
should be. it's live code. seems @sparhami added it

Reply:
I tried a few different browsers (Firefox, Chrome on Linux) that have vertical scrollbars that take up some width and confirmed it worked. I also tried browsers that have overlaying scrollbars and confirmed it returned zero.

I can't remember if I tested IE, but I know it supports `clientWIdth`, which is defined as the width excluding the scrollbar.

Reply:
Test Safari please, it's uses a different root scroller.

Reply:
Safari has an overlaid scrollbar, so it has no scrollbar width. I tested that (as well as iOS).